--- origsrc/xbase-2.0.0/xbase/Makefile.am	2001-06-26 17:36:43.000000000 -0500
+++ src/xbase-2.0.0/xbase/Makefile.am	2008-09-03 01:36:14.111605300 -0500
@@ -56,7 +56,7 @@
 	makefile.g95 \
 	xbase.ide
 
-libxbase_la_LDFLAGS = -version-info 0:0:0 -release 2.0
+libxbase_la_LDFLAGS = -version-info 0:0:0 -release 2.0 -no-undefined
 libxbase_la_LIBADD = 
 
 MAINTAINERCLEANFILES = Makefile.in stamp-h.in xbconfig.h.in
