--- origsrc/xbase-2.0.0/docs/doxygen.cfg.in	2000-12-12 00:32:00.000000000 -0600
+++ src/xbase-2.0.0/docs/doxygen.cfg.in	2008-09-03 01:46:55.015316500 -0500
@@ -268,7 +268,7 @@
 # directories like "/usr/src/myproject". Separate the files or directories 
 # with spaces. 
 
-INPUT                  = ../xbase
+INPUT                  = @top_builddir@/xbase @top_srcdir@/xbase
 
 # If the value of the INPUT tag contains directories, you can use the 
 # FILE_PATTERNS tag to specify one or more wildcard pattern (like *.cpp 
