xbase/ntx.h:221: error: extra qualification ‘xbNtx::’ on member ‘AllocKeyBufs’

--- origsrc/xbase-2.0.0/xbase/ntx.h	2000-11-10 13:04:17.000000000 -0600
+++ src/xbase-2.0.0/xbase/ntx.h	2010-06-10 18:19:36.293808500 -0500
@@ -218,7 +218,7 @@ class XBDLLEXPORT xbNtx : public xbIndex
    xbShort  KeyExists( char * Key ) { return FindKey( Key, strlen( Key ), 0 ); }
    xbShort  KeyExists( xbDouble );
 
-   xbShort xbNtx::AllocKeyBufs(void);
+   xbShort AllocKeyBufs(void);
 
    virtual void GetExpression(char *buf, int len);
 };
